packages/svelte/src/internal/client/reactivity/effects.js

```diff
@@ -311,16 +311,17 @@ export function execute_effect_teardown(effect) {
 
 /**
  * @param {import('#client').Effect} effect
+ * @param {boolean} skip_remove_dom
  * @returns {void}
  */
-export function destroy_effect(effect) {
+export function destroy_effect(effect, skip_remove_dom) {
```

Comment:
Should the argument (here and the other function) be `remove_dom` instead, i.e. the inverse? Reads a bit better

Reply:
@dummdidumm The size difference between them is something 14 bytes when I checked the min bundle size.

Reply:
I made the change anyway